### ISSUE TITLE
Pset 2

### ISSUE THREAD

**Issue by kameeko** [alanedelman/18.337_2016#4]
In Pset 2, after I define SymArrow2, and overload the size and getindex methods to handle SymArrow2 types, I test the \* method with a random matrix and vector of complex numbers. In version 0.4.6, I get the message:

```
LoadError: MethodError: `A_mul_B!` has no method matching A_mul_B!(::Array{Complex{Float64},1}, ::SymArrow2{Complex{Float64}}, ::Array{Complex{Float64},1})
```

but in version 0.5.0-rc4 (accessed via JuliaBox), it runs without complaint. Am I defining SymArrow2 or the overloaded methods incorrectly, or are the fall-back methods that we're intended to use different between the two versions?

**Comment by dpsanders** [comment]
Hi,

Apparently this indeed works only on 0.5.

Best
David

**Comment by kameeko** [comment]
I installed Julia using the Ubuntu-specific installation instructions, so I have v0.4.6 on my computer. Is there a way to have multiple versions of Julia on the same computer, so that I can switch between different versions without being online (using JuliaBox)?

**Comment by dpsanders** [comment]
Yes you can have multiple versions: you can use the generic Linux binary and give it a different name, e.g. julia5.

**Comment by kameeko** [comment]
Woah, and Jupyter knows about both even though they are in completely different places. This is sweet.

**Comment by dpsanders** [comment]
Yes ;)

**Comment by alanedelman** [comment]
i also was kind of surprised how well that works
